src/zenml/zen_server/feature_gate/zenml_cloud_feature_gate.py

```diff
@@ -60,7 +60,7 @@ class RawUsageEvent(BaseModel):
 
 
 class ZenMLCloudFeatureGateInterface(FeatureGateInterface):
-    """ZenML Cloud Feature Gate implementation."""
+    """ZenML Pro Feature Gate implementation."""
```

Comment:
I think this is the only valid mention, but I left it on purpose as we can't rename the class without more effort.